bin/disco_aws.py

```diff
@@ -293,6 +296,8 @@ def run():
             if args.productline or args.all:
                 productline = instance.tags.get("productline", u"unknown")
                 line += u" {0:<15}".format(productline if productline != u"unknown" else u"-")
+            if args.securitygroup or args.all:
+                line += u" {0:15}".format(instance.groups[0].name)
```

Comment:
Do you always want the first group if there are more than one item in `instance.groups`?